src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -498,8 +495,11 @@ public Object readResolve() throws ObjectStreamException {
             this.startTls = true;
         }
 
+        //requireTls can be  null, to avoid null-pointer exception, set flag as false.
+        boolean requireTlsFlag = requireTLS == null ? false : requireTLS;
```

Comment:
This will cause a null pointer exception if not set to false.

Reply:
Why not simply invoking the method below with `Boolean.valueOf(requireTls)` ?